### PR TITLE
Introduced tests and documentation for the Italian fiscalization

### PR DESCRIPTION
**Pull request by abdallahbeshi**
## Description

Introduced tests and documentation for the Italian fiscalization.

## Type of change
- [ ] Bug fix.
- [ ] Feature.
- [x] Consolidation.

## Checklist
- [ ] Is breaking change.
- [x] Documentation updated.
- [x] Tests included.

### CODE REVIEW

**Review by oleg-sokolansky — APPROVED** [review]

**Comment by oleg-sokolansky** [comment]
I guess it is less relevant to PR itself, but why did you split workflows by os?

File: src/Italy/Mews.Fiscalizations.Italy.Tests/UniwixClientTests.cs [hunk 1]
```diff
@@ -0,0 +1,218 @@
...
+        [Test]
+        public async Task SendInvoiceSucceeds()
+        {
+            var client = GetUniwixClient();
+            var invoiceNumber = new Random().Next(1, 9999).ToString();
+            var result = await client.SendInvoiceAsync(new ElectronicInvoice
+            {
+                Version = VersioneSchemaType.FPR12,
+                Header = GetInvoiceHeader(invoiceNumber),
+                Body = new[] { GetInvoiceBody(invoiceNumber) }
```
Comment (KaliCZ) [inline src/Italy/Mews.Fiscalizations.Italy.Tests/UniwixClientTests.cs:34]: Won't this complain about duplicate invoices? Can't we make it like in other libraries, where the date is present in the invoice number?

File: .github/workflows/build-and-test-all-linux.yml [hunk 2]
```diff
@@ -55,4 +55,7 @@ jobs:
         hungarian_signing_key: ${{secrets.hungarian_signing_key}}
         hungarian_tax_payer_id: ${{secrets.hungarian_tax_payer_id}}
         hungarian_encryption_key: ${{secrets.hungarian_encryption_key}}
+
+        italian_username: ${{secrets.italian_username}}
+        italian_password: ${{secrets.italian_password}}
```
Comment (KaliCZ) [inline .github/workflows/build-and-test-all-linux.yml:60]: How come the builds are green when I didn't  put the secrets there yet? Did somebody else do it?

Comment (abdallahbeshi) [reply .github/workflows/build-and-test-all-linux.yml:60]: @marektresnak did.

File: src/Italy/Mews.Fiscalizations.Italy.Tests/UniwixClientTests.cs (hunk 1 above)
Comment (abdallahbeshi) [reply src/Italy/Mews.Fiscalizations.Italy.Tests/UniwixClientTests.cs:34]: Seems like it won't in the test environment as the state will be always "Uploading file..." (which is pending).

**Review by KaliCZ — APPROVED** [review]

**Pull request merged**